components/brave_new_tab_ui/components/default/searchPromotion/search-box.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react'
```

Comment:
Not 100% sure on our naming conventions, but it looks like most of the other files are `camelCase` rather than `hyphen-case`.